test/test_h_transport_problem.py

```diff
@@ -617,12 +617,14 @@ def test_run_in_steady_state():
     """Test that the run method works in steady state"""
     # BUILD
     my_vol = F.VolumeSubdomain1D(id=1, borders=[0, 4], material=dummy_mat)
+    H = F.Species("H")
     my_model = F.HydrogenTransportProblem(
         mesh=test_mesh,
         temperature=500,
+        sources=[F.Source(value=1.0, volume=my_vol, species=H)],
         settings=F.Settings(atol=1e-10, rtol=1e-10, transient=False),
         subdomains=[my_vol],
-        species=[F.Species("H")],
+        species=[H],
```

Comment:
why?

Reply:
Was getting a bug locally, but found the issue. will change it back